cv32/sim/Common.mk

```diff
@@ -199,7 +199,7 @@ FIRMWARE_UNIT_TEST_OBJS   =  	$(addsuffix .o, \
 ###############################################################################
 # The sanity rule runs whatever is currently deemed to be the minimal test that
 # must be able to run (and pass!) prior to generating a pull-request.
-sanity: hello-world
+sanity: hello_world
```

Comment:
Hi @masgia, its probably not a good idea to change this: the CI config files invoke these targets and this could break them.   In addition, several Contributors have their own scripts that invoke these targets.